packages/aws-cdk-lib/aws-rds/lib/cluster.ts

```diff
@@ -972,3 +982,120 @@ function createInstances(cluster: DatabaseClusterNew, props: DatabaseClusterBase
 function databaseInstanceType(instanceType: ec2.InstanceType) {
   return 'db.' + instanceType.toString();
 }
+
+type UnwantedDatabaseCluserProps = 'instances' | 'instanceIdentifierBase' | 'instanceUpdateBehavior' |  'backtrackWindow' | 'port';
+export interface MultiAZDatabaseClusterProps extends Omit<DatabaseClusterProps, UnwantedDatabaseCluserProps> {
```

Comment:
The only `StorageType` supported for [Multi AZ Database Clusters is Provisioned IOPS storage (Io1)](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/multi-az-db-clusters-concepts.html#multi-az-db-clusters-concepts.Limitations)

So rather than being a passed in prop it's handled internally
https://github.com/aws/aws-cdk/blob/2db7f768a4dc15aa8dea297cc22d3239b33fc7cd/packages/aws-cdk-lib/aws-rds/lib/cluster.ts#L1072-L1073

`DBClusterInstanceClass` is currently handled by the `instanceProps` prop. 
https://github.com/aws/aws-cdk/blob/2db7f768a4dc15aa8dea297cc22d3239b33fc7cd/packages/aws-cdk-lib/aws-rds/lib/cluster.ts#L1058
I have gone back and forth about for `MutliAZDatabaseCluster` the instance props should be directly on `MultiAZDatabaseClusterProps` or not. I went with using `instanceProps` for consistency, but could easily get moved to having them be in `MultiAZDatabaseClusterProps`.